### DISCUSSION TITLE
Gauge Severity Low and High

### DISCUSSION THREAD

**Discussion by kreene1987** [home-assistant/frontend#8357] [Dashboard]
I'd like the ability to define a gauge for both high and low severity. For example a temperature sensor:

Low Red:1-60
Low Yellow: 61
Low Green: 66
High Green: 74
High Yellow: 78
High Red: 80

Such that on my lovelace dashboard I can immediately see any temperatures that are out of ideal range quickly, both high AND low.

**Comment by bucydan32** [comment]
+1 Id like to be able to use it to monitor battery voltages on my solar battery bank

**Comment by karwosts** [comment]
This has been possible for some time, closing.
